### PR TITLE
Product overview

### PR DESCRIPTION
**Pull request by robertvgarcia**

### CODE REVIEW

**Review by Flaws77 — APPROVED** [review]
Seems fine to me

File: client/src/components/ProductOverview/ImageGallery.jsx [hunk 1]
```diff
@@ -1,12 +1,22 @@
 import React from 'react';
+import Thumbnails from './Thumbnails'
 import s from './style.css';
 import axios from 'axios'
 
-const ImageGallery = () => {
-
+const ImageGallery = ({ styleInfo }) => {
+  console.log(styleInfo)
```
Comment (keewook2) [inline client/src/components/ProductOverview/ImageGallery.jsx:7]: remove console.log please

File: client/src/components/ProductOverview/ProductInfo.jsx [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 import s from './style.css';
 
-const ProductInfo = () => {
-
+const ProductInfo = ({ productInfo }) => {
   return (
     <div id={s.info}>
-      <h1>Product Info</h1>
+      <p>
+        <span>&#9733;&#9733;&#9733;&#9733;&#9733; </span>
```
Comment (keewook2) [inline client/src/components/ProductOverview/ProductInfo.jsx:8]: Better use Star components to show average star ratings!, and if hardcode is needed, you can copy and paste star unicode directly.

File: client/src/components/ProductOverview/ProductInfo.jsx [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+        <a href="#"><small>Read all reviews</small></a>
```
Comment (keewook2) [inline client/src/components/ProductOverview/ProductInfo.jsx:9]: Using <Link> in react-router-dom is recommended becuase if you use <a> tag it'll lose state and show lower performance.

**Review by keewook2 — CHANGES_REQUESTED** [review]

File: client/src/components/ProductOverview/ProductInfo.jsx (hunk 3 above)
Comment (keewook2) [reply client/src/components/ProductOverview/ProductInfo.jsx:9]: `<a>` tag

**Pull request merged**